cmd/tendermint/commands/lite.go

```diff
@@ -99,50 +97,33 @@ func runProxy(cmd *cobra.Command, args []string) error {
 
 	chainID = args[0]
 
-	logger.Info("Creating HTTP client for primary...", "addr", primaryAddr)
-	rpcClient, err := rpcclient.NewHTTP(primaryAddr, "/websocket")
-	if err != nil {
-		return errors.Wrapf(err, "http client for %s", primaryAddr)
-	}
-	primary := httpp.NewWithClient(chainID, rpcClient)
-
-	// TODO: use NewNetClient once we have it
-	addrs := strings.Split(witnessesAddrs, ",")
-	witnesses := make([]provider.Provider, len(addrs))
-	for i, addr := range addrs {
-		p, err := httpp.New(chainID, addr)
-		if err != nil {
-			return errors.Wrapf(err, "http provider for %s", addr)
-		}
-		witnesses[i] = p
-	}
-
 	logger.Info("Creating client...", "chainID", chainID)
+	addrs := strings.Split(witnessesAddrs, ",")
```

Comment:
```suggestion
	witnesses := strings.Split(witnessesAddrs, ",")
```

addrs is too abstract